mapper_test.go

```diff
@@ -632,8 +638,7 @@ func TestMapperPlaceHolder(t *testing.T) {
 	assert.Contains(t, b.String(), "--flag=/a/b/c")
 }
 
-type testMapperWithPlaceHolder struct {
-}
+type testMapperWithPlaceHolder struct{}
```

Comment:
oops. this snuck in from gofumpt.
I can revert this line if you care about the difference.